src/Pages/SignIn/SignIn.js

```diff
@@ -29,14 +29,14 @@ export default function SignIn() {
     setPassword(data.get("password"));
   };
 
-  const getUserToken = () => {
+  const getUserToken = async () => {
     const url = "/api/v1/auth/login";
     const data = {
       email: email,
       password: password,
     };
 
-    axios
+    await axios
```

Comment:
Again, you do not need await if you are using .then.  You could have either await or .then, but you don't want to have both.

Reply:
Removed await syntax